examples/advanced/src/index.html

```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
+<script type="module">
+  import './app.tsx';
+</script>
```

Comment:
Pretty unusual, maybe replace it with more common 
`<script type="module" src="./app.tsx"></script>` ?